support-frontend/assets/pages/digital-subscription-checkout/components/addressForm.jsx

```diff
@@ -0,0 +1,109 @@
+// @flow
+
+import React from 'react';
+import { compose } from 'redux';
+
+// helpers
+import { firstError, type FormError } from 'helpers/subscriptionsForms/validation';
+import { isPostcodeOptional } from 'pages/digital-subscription-checkout/helpers/validation';
+import { auStates, caStates, countries, type IsoCountry, usStates } from 'helpers/internationalisation/country';
+
+// hocs
+import { withLabel } from 'hocs/withLabel';
+import { withError } from 'hocs/withError';
+import { asControlled } from 'hocs/asControlled';
+import { canShow } from 'hocs/canShow';
+
+// components
+import { Input } from 'components/forms/input';
+import { Select } from 'components/forms/select';
+import { sortedOptions } from 'components/forms/customFields/sortedOptions';
+
+const InputWithLabel = withLabel(Input);
```

Comment:
Can you extract these to a new module in `components/subscriptionsCheckouts` and import them into this file and the checkout form? It would be good to use them in the paper checkout form as well because they are also declared in there, but up to you if you want to leave that for another PR

Reply:
I'm not sure if doing so kinda goes against the spirit of the HoCs? i like having them defined inline because it makes it easy to know exactly what they are, if we unify them in one file we could as well export a single input component that takes labels and errors as optional props?